cPP_DSC.adoc

```diff
@@ -4019,203 +4015,391 @@ The dependencies between SFRs implemented by the TOE are addressed as follows.
 |Rationale Statement
 
 |FCS_CKM.1 
-|[FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_COP.1] 
+|[_FCS_CKM.1/AKG_, _FCS_CKM.1/SKG_, FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7, _FCS_CKM_EXT.8_ or FCS_COP.1] 
+
+FCS_CKM.3
 
 [FCS_RBG.1 or FCS_RNG.1]
 
-FCS_CKM.6 
-|FCS_CKM.2, FCS_CKM.6, FCS_CKM_EXT.7, FCS_COP.1 and FCS_RBG.1 are required by the PP.
+FCS_CKM.6
+|FCS_CKM.1/AKG - (selection-based) included
+
+FCS_CKM.1/SKG - (selection-based) included
+
+FCS_CKM.2 - included
+
+FCS_CKM.3 is satisfied by FAU_STG_EXT.1 which controls access to keys - included
+
+FCS_CKM_EXT.7 - included
+
+FCS_CKM_EXT.8 - (selection-based) included
+
+FCS_COP.1 - included
+
+FCS_RBG.1 - included
+
+FCS_RNG.1 is satisfied by FCS_RBG.1 - included
+
+FCS_CKM.6 - included
 
 |FCS_CKM.2
-|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1 or FCS_CKM.5] 
+|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, FCS_CKM.5 or _FCS_COP.1_] 
+|FDP_ITC.1 is satisfied by FDP_ITC_EXT.1 related to a mechanism by which key data can be imported into the TSF - included
+
+FDP_ITC.2 is satisfied by FDP_ITC_EXT.2 related to a mechanism by which key data can be imported into the TSF - included
+
+FCS_CKM.1 - included
```

Comment:
Aren't FDP_ITC.1, FDP_ITC.2, and FCS_CKM.1 selection-based too?

Reply:
Unless selection-based here refers to the SFR in the DSC as opposed to a selection in FCS_CKM.2

Reply:
selection-based is that the SFR itself is selection-based, not what is a selection within the SFR. This is basically what the Network Device cPP looked like (though they said "selection-based SFR" and I left off the SFR).